frontend/src/metabase-lib/v1/expressions/helper-text-strings.ts

```diff
@@ -1412,19 +1416,54 @@ function isArgsExpression(x: unknown): x is ["args", Expression[]] {
   return Array.isArray(x) && x[0] === "args";
 }
 
-const getHelpExample = ({ name, args = [] }: HelpTextConfig): Expression => {
-  const parameters: Expression[] = [];
+/**
+ * Build the expression example as a Lib.ExpressionParts manually.
+ * This is necessary because we don't have a query to refer to in the examples.
+ *
+ * TODO: can we approach this differently?
+ */
+const getHelpExample = ({
```

Comment:
@romeovs I think that we should revert the change where we used the real formatter for examples. It introduces more complexity than it solves.